src/mmw/js/src/projects/views.js

```diff
@@ -0,0 +1,160 @@
+"use strict";
+
+var Marionette = require('../../shim/backbone.marionette'),
+    App = require('../app'),
+    router = require('../router').router,
+    models = require('../modeling/models'),
+    modalModels = require('../core/modals/models'),
+    modalViews = require('../core/modals/views'),
+    containerTmpl = require('./templates/container.html'),
+    rowTmpl = require('./templates/projectRow.html');
+
+var ProjectsView = Marionette.LayoutView.extend({
+    template: containerTmpl,
+
+    ui: {
+        newProject: '.new-project'
+    },
+
+    events: {
+        'click @ui.newProject': 'newProject'
+    },
+
+    regions: {
+        rowContainer: '.project-rows-container'
+    },
+
+    onShow: function() {
+        var self = this,
+            projects = new models.ProjectCollection();
+        projects
+            .fetch()
+            .done(function() {
+                self.rowContainer.show(new ProjectRowsView({
+                    collection: projects
+                }));
+            });
+    },
+
+    newProject: function() {
+        App.currentProject = null;
```

Comment:
These are all the places where we set `currentProject` to `null`:

 * [`AnalyzeController.analyzePrepare`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/analyze/controllers.js#L46)
 * [`core/testUtils:resetApp`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/testUtils.js#L28)
 * [`ItsiEmbed.loadInteractive`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/itsiEmbed.js#L48)
 * [`ModelingController.projectDraw`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/controllers.js#L202)

While we are clearing the `App.currentProject` (and notably not clearing `App.projectNumber` which I believe was a later addition), we're not always clearing the AoI, and I don't think we want to in all these cases. I agree that clearing the project and AoI should be a function on `App`, but I'm not sure if it will be able to replace all these occurrences.